scripts/ci/publish.ts

```diff
@@ -20,9 +20,14 @@ const packagesDir = join(process.cwd(), C.PACKAGES_DIRNAME)
 const packagesList = sortPackagesByPreference(packagesDir)
 
 ;(async () => {
-  const version = await doAskVersionToPublish()
-  const confirmation = await doAskForConfirmation(version)
-  if (!confirmation) return
+  let version = ''
```

Comment:
move this to a getVersionAndConfirmation()

Reply:
refactored, also added support for publishing betas